misk-hibernate/src/test/kotlin/misk/hibernate/pagination/RealPagerTest.kt

```diff
@@ -10,19 +10,23 @@ import misk.hibernate.Query
 import misk.hibernate.Transacter
 import misk.hibernate.load
 import misk.hibernate.or
+import misk.jdbc.DataSourceType
+import misk.jdbc.ScaleSafetyChecks
 import misk.testing.MiskTest
 import misk.testing.MiskTestModule
 import org.assertj.core.api.Assertions.assertThat
 import org.hibernate.SessionFactory
 import org.junit.jupiter.api.Test
+import java.sql.Timestamp
+import java.time.Instant
 import java.time.LocalDate
 import javax.inject.Inject
 
 @MiskTest(startService = true)
 class RealPagerTest {
 
   @MiskTestModule
-  val module = MoviesTestModule()
+  val module = MoviesTestModule(DataSourceType.MYSQL)
```

Comment:
The default `VITESS_MYSQL` data source doesn't like `SET GLOBAL`, used to enable the general query log